dot/network/connmgr.go

```diff
@@ -144,18 +142,6 @@ func (cm *ConnManager) Disconnected(_ network.Network, c network.Conn) {
 	}
 }
 
-// OpenedStream is called when a stream is opened
-func (cm *ConnManager) OpenedStream(_ network.Network, s network.Stream) {
-	logger.Tracef("Stream opened with peer %s using protocol %s",
-		s.Conn().RemotePeer(), s.Protocol())
-}
-
-// ClosedStream is called when a stream is closed
-func (cm *ConnManager) ClosedStream(_ network.Network, s network.Stream) {
-	logger.Tracef("Stream closed with peer %s using protocol %s",
-		s.Conn().RemotePeer(), s.Protocol())
-}
```

Comment:
Can't we hook in the stream opening and closing to trace logs anymore?
Or is it done by the Connected and Disconnected methods now?
Not a big deal, but just curious :thinking:

Reply:
:thinking: Any idea how we can 'get a notification that the swarm opened a new stream'? :thinking:

Reply:
the stream handler functions (`handleStream`) are invoked whenever a new stream is opened, and when they close you will get a "stream reset" error message when trying to read from them I believe